conans/client/conf/__init__.py

```diff
@@ -159,6 +159,16 @@
                     threads: [None]
                     exceptions: [None]
 
+    # opt-in for stating which versions of compilers are compatible with other versions
```

Comment:
This is to be implemented in the ``compatibility.py`` plugin in 2.0.

Reply:
> This is very unlikely that it will fly for Conan 1.X scope.

Is there any particular reason for it? This feature works pretty well with the current 1.X set ups and does not break compatibility with anything in 1.X.

Reply:
Yes, every single new feature requires way more effort than it seems in the PR. The later derivatives, questions, support tickets, bug reports, fixes, documentation, ux issues, discussions of approaches with the team, etc, are like 95% of the work to do, the PR with the initial feature is just the tip of the iceberg. As we are already in beta.2, we don't have time or resources to do this in 1.X scope, all of our efforts goes to 2.0, and just backport to 1.X the necessary things for the upgrade.